frontend/public/components/utils/_copy-to-clipboard.scss

```diff
@@ -3,12 +3,10 @@
   position: relative;
 }
 
-.co-copy-to-clipboard__btn {
-  position: absolute;
+.pf-c-clipboard-copy__group-copy.co-copy-to-clipboard__btn {
```

Comment:
Do you need the extra specificity **and** the `!important`?

Reply:
removed `!important`